win/tty/termcap.c

```diff
@@ -1166,7 +1176,12 @@ kill_hilite()
         if (hilites[c | BRIGHT] && hilites[c | BRIGHT] != nh_HI)
             free((genericptr_t) hilites[c | BRIGHT]), hilites[c | BRIGHT] = 0;
     }
-    return;
+
+    if (hilites[CLR_BLACK]) {
```

Comment:
nice catch on the dealloc.

Reply:
(Not sure if that's clear already, but: This dealloc is only necessary / correct given the other changes in this commit. Before, `hilites[CLR_BLACK]` was always `NULL` for the macro definitions we use.)